slides/quickstart/index.html

```diff
@@ -104,25 +63,18 @@
       function listSlides() {
         gapi.client.slides.presentations.get({
           presentationId: '1EAYk18WDjIG-zp_0vLm3CsfQh_i8eXc67Jo2O9C6Vuc'
-        }).then(function(response) {
-          var presentation = response.result;
-          var length = presentation.slides.length;
-          appendPre('The presentation contains ' + length + ' slides:');
-          for (i = 0; i < length; i++) {
-            var slide = presentation.slides[i];
-            appendPre('- Slide #' + (i + 1) + ' contains ' +
-                slide.pageElements.length + ' elements.')
-          }
-        }, function(response) {
-          appendPre('Error: ' + response.result.error.message);
+        }).then((response) => {
+          const presentation = response.result;
+          const length = presentation.slides.length;
+          appendPre(`The presentation contains ${length} slides:`);
+          presentation.slides.map((slide, i) => {
+            appendPre(`- Slide #${(i + 1)} contains ${slide.pageElements.length} elements.`);
+          });
+        }, (response) => {
+          appendPre(`Error: ${response.result.error.message}`);
         });
       }
-
-    </script>
-
-    <script async defer src="https://apis.google.com/js/api.js"
```

Comment:
This version comes from the official docs for this library:

https://developers.google.com/api-client-library/javascript/samples/samples#loading-the-library-asychronously

It's needed for compatibility with older versions if IE.

Reply:
I think those docs are not up-to-date though.
- `onreadystatechange` was [removed in IE11](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/dev-guides/bg182625(v=vs.85)#legacy-api-additions-changes-and-removals).
- `this.onload=function(){};` I think this is usually used to prevent an XSS attacks. I don't think it's needed here though.

Reply:
As mentioned in the docs, it's for compatibility with older versions of IE, not IE11. Our samples should adhere to the official samples, but you can reach out to the library owners to verify that their samples are up-to-date.